cypress-tests/cypress/support/commands.js

```diff
@@ -250,7 +252,11 @@ Cypress.Commands.add("confirmCallTest", (confirmBody, details, confirm, globalSt
         throw new Error(`Unsupported authentication type: ${authentication_type}`);
       }
     }
-    else {
+    else if(response.body.error.type === "invalid_request") {
+      if(response.body.error.message === "Payment method type not supported") {
+        expect(response.body.error).to.deep.equal(Errors.paymentMethodErrors[globalState.get("connectorId")].paymentMethodUnsupportedError);
+      }
+    } else {
```

Comment:
Can we move this to `manual` else if block, since we are explicitly saying Manual Capture for trustpay is not supported

Reply:
done